### PR TITLE
Removed test migrations. Tests will now run without them.

### PR DESCRIPTION
**Pull request by czpython**

### CODE REVIEW

File: testserver.py [hunk 1]
```diff
@@ -24,6 +27,19 @@ def noop_gettext(s):
...
+
+class DisableMigrations(object):
+
+    def __contains__(self, item):
+        return True
+
+    def __getitem__(self, item):
+        return 'notmigrations'
+
+    def update(self, whatever):
```
Comment (czpython) [inline testserver.py]: :trollface:

Comment (vxsx) [inline testserver.py]: :trollface:

Comment (FinalAngel) [inline testserver.py]: :trollface:

**Comment by FinalAngel** [comment]
Can't really approve or deny :D

**Pull request merged**